adapters-2.0/src/test/java/com/flipkart/fdp/ml/adapter/DecisionTreeClassificationModelBridgeTest.java

```diff
@@ -0,0 +1,57 @@
+package com.flipkart.fdp.ml.adapter;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.spark.ml.classification.DecisionTreeClassificationModel;
+import org.apache.spark.ml.classification.DecisionTreeClassifier;
+import org.apache.spark.ml.linalg.SparseVector;
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.junit.Test;
+
+import com.flipkart.fdp.ml.export.ModelExporter;
+import com.flipkart.fdp.ml.importer.ModelImporter;
+import com.flipkart.fdp.ml.transformer.DecisionTreeTransformer;
+
+
+public class DecisionTreeClassificationModelBridgeTest extends SparkTestBase {
+
+
+    @Test
+    public void testDecisionTreeRegressionPrediction() {
```

Comment:
Please add a test using this in pipeline as well. You can refer 1.6 test for the same.